utils/build_swift/tests/expected_options.py

```diff
@@ -768,7 +768,7 @@ class BuildScriptImplOption(_BaseOption):
     ChoicesOption('--swift-analyze-code-coverage',
                   choices=['false', 'not-merged', 'merged']),
     ChoicesOption('--android-arch',
-                  choices=['armv7', 'aarch64']),
+                  choices=['armv7', 'aarch64', 'x86_64']),
```

Comment:
This never triggered before because this just checks if it's a subset, so updating to all three arches supported now.